localstack/services/acm/provider.py

```diff
@@ -55,6 +56,32 @@ def describe(describe_orig, self):
     return result
 
 
+@singleton_factory
+def patch_cert_bundle_pickling():
+    """Apply patches to pickle CertBundle base models. It prevents the "cannot pickle 'builtins.Certificate'" errors"""
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.x509 import load_pem_x509_certificate
+    from moto.acm.models import CertBundle
+
+    def cert_set_state(self, state, *args, **kwargs):
```

Comment:
I think you could also use the decorator in `localstack.utils.patch` for this:
```
@patch(CertBundle.__setstate__)
def cert_set_state(self, state, *args, **kwargs):
  ...
```

Reply:
Could our decorator be used this way here? `acm.moto.CertBundle` does not have `__setstate__`

Reply:
Great catch - I upvoted @alexrashed 's comment a bit too quickly, :) but there was a reason why we didn't use `@patch` previously! (as this code is copied/migrated from -ext)

Reply:
👍 thanks for the clarification @whummer